sdk/storage/azure-storage-blob/azure/storage/blob/_deserialize.py

```diff
@@ -32,6 +32,29 @@ def deserialize_blob_properties(response, obj, headers):
     return blob_properties
 
 
+def deserialize_ors_policies(response):
+    # For source blobs (blobs that have policy ids and rule ids applied to them),
+    # the header will be formatted as "x-ms-or-<policy_id>_<rule_id>: {Complete, Failed}".
+    # The value of this header is the status of the replication.
+    or_policy_status_headers = {key: val for key, val in response.headers.items()
+                                if key.startswith('x-ms-or') and key != 'x-ms-or-policy-id'}
+
+    parsed_result = {}
+
+    for key, val in or_policy_status_headers.items():
+        policy_and_rule_ids = key[len('x-ms-or-'):].split('_')
```

Comment:
Do we need the `len` operation? This could just be a constant value with a comment to state it's meaning

Reply:
Also - is it possible for the policy or rule ID themselves to container underscores?

Reply:
Sounds good! No there's no underscore in the policy/rule ID